plugins/extractors/bigquery/bigquery.go

```diff
@@ -444,15 +444,21 @@ func (e *Extractor) buildAsset(ctx context.Context, t *bigquery.Table, md *bigqu
 		}
 	}
 
-	table, err := anypb.New(&v1beta2.Table{
-		Columns:       e.buildColumns(ctx, md.Schema, md),
-		PreviewFields: previewFields,
-		PreviewRows:   previewRows,
-		Profile:       tableProfile,
-		Attributes:    utils.TryParseMapToProto(attributesData),
-		CreateTime:    timestamppb.New(md.CreationTime),
-		UpdateTime:    timestamppb.New(md.LastModifiedTime),
-	})
+	tableData := &v1beta2.Table{
+		Columns:    e.buildColumns(ctx, md.Schema, md),
+		Profile:    tableProfile,
+		Attributes: utils.TryParseMapToProto(attributesData),
+		CreateTime: timestamppb.New(md.CreationTime),
+		UpdateTime: timestamppb.New(md.LastModifiedTime),
+	}
+
+	maxPreviewRows := e.config.MaxPreviewRows
+	if maxPreviewRows != -1 {
```

Comment:
Can you add documenation for this field and value in the struct and also in readme - https://github.com/goto/meteor/blob/main/plugins/extractors/bigquery/bigquery.go#L52